documentation/manual/aggregation.md

```diff
@@ -81,4 +81,4 @@ Chaining together more than two aggregateBy methods is also possible, which resu
 
 ```java
 OSHDBCombinedIndex<OSHDBCombinedIndex<IndexType1, IndexType2>, IndexType3>
-```
\ No newline at end of file
+```
```

Comment:
what's the change in this line?

Reply:
probably just a line ending (before the end of the file) was added by an editor